Makefile

```diff
@@ -1,19 +1,13 @@
-all: bounties sunodo-sdk-image image
+all: bounties image
 
 image:
 	sunodo build
 
 bounties:
 	$(MAKE) -C tests/bounties
 
-sunodo-sdk-image: linux.bin
-	docker build --tag sunodo/sdk:0.2.0-sandboxing --file sunodo-sdk.Dockerfile --progress plain .
-
-linux.bin:
-	wget -O linux.bin https://github.com/crypto-bug-hunters/bugless/releases/download/kernel/linux.bin
-
 test:
```

Comment:
This test will probably fail because of missing Lua packages as stated.

Were tests working before this PR? I don't know if after all the recent changes the Lua test suite stopped to be maintained or working. If it was not already working maybe you can merge as it is and create an issue about fixing test suite later.

Reply:
I've fixed the tests in #83